lib/peer/distributed-peer.js

```diff
@@ -3,6 +3,8 @@ const Messages = require('../messages.js')
 const MessageFactory = require('../message-factory.js')
 const debug = require('debug')('slsk:peer:distributed:i')
 
+let stack = require('../stack')
```

Comment:
is this needed?

Reply:
I mean, I was curious about this addition, but in fact seems to be already defined in master branch, maybe it's just a github flaw

Reply:
Interesting. I see that as well. I suppose it should be left in this PR though, otherwise it'll get taken out.